pkg/executor/executor.go

```diff
@@ -384,14 +392,14 @@ func StartExecutor(ctx context.Context, clientGen crd.ClientGeneratorInterface,
 		informerFactory.Start(ctx.Done())
 	}
 
-	api, err := MakeExecutor(ctx, logger, cms, fissionClient, executorTypes,
+	api, err := MakeExecutor(ctx, logger, mgr, cms, fissionClient, executorTypes,
 		fissionInformers...,
 	)
 	if err != nil {
 		return err
 	}
 
-	utils.CreateMissingPermissionForSA(ctx, kubernetesClient, logger)
+	utils.CreateMissingPermissionForSA(ctx, kubernetesClient, logger, mgr)
```

Comment:
We shouldnt pass mgr to this function. Probably we can make function sync and can call via mgr.Add

Reply:
done

Reply:
done